<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>GTATitleBasedExecution</class>
	<widget class="QWidget" name="GTATitleBasedExecution">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>751</width>
				<height>609</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<property name="windowTitle">
			<string>Title based execution</string>
		</property>
		<property name="windowIcon">
			<iconset resource="../resource.qrc">
				<normaloff>:/images/forms/img/GTAAppIcon.png</normaloff>:/images/forms/img/GTAAppIcon.png</iconset>
		</property>
		<layout class="QGridLayout" name="gridLayout">
			<item row="0" column="0">
				<layout class="QHBoxLayout" name="horizontalLayout">
					<item>
						<widget class="QLabel" name="ProcNameLbl">
							<property name="sizePolicy">
								<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
									<horstretch>0</horstretch>
									<verstretch>0</verstretch>
								</sizepolicy>
							</property>
							<property name="text">
								<string/>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QToolButton" name="SelectAllTB">
							<property name="text">
								<string>...</string>
							</property>
							<property name="icon">
								<iconset resource="../resource.qrc">
									<normaloff>:/images/forms/img/SelectAll.png</normaloff>:/images/forms/img/SelectAll.png</iconset>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QToolButton" name="DeSelectAllTB">
							<property name="text">
								<string>...</string>
							</property>
							<property name="icon">
								<iconset resource="../resource.qrc">
									<normaloff>:/images/forms/img/DeselectAll.png</normaloff>:/images/forms/img/DeselectAll.png</iconset>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QToolButton" name="ExpandAllTB">
							<property name="text">
								<string>...</string>
							</property>
							<property name="icon">
								<iconset resource="../resource.qrc">
									<normaloff>:/images/forms/img/Expandall.png</normaloff>:/images/forms/img/Expandall.png</iconset>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QToolButton" name="CollapseAllTB">
							<property name="text">
								<string>...</string>
							</property>
							<property name="icon">
								<iconset resource="../resource.qrc">
									<normaloff>:/images/forms/img/Collapseall.gif</normaloff>:/images/forms/img/Collapseall.gif</iconset>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QToolButton" name="ExecuteTB">
							<property name="text">
								<string>...</string>
							</property>
							<property name="icon">
								<iconset resource="../resource.qrc">
									<normaloff>:/images/forms/img/Launch.png</normaloff>:/images/forms/img/Launch.png</iconset>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item row="1" column="0">
				<widget class="QTreeView" name="ElementTV"/>
			</item>
		</layout>
	</widget>
	<resources>
		<include location="../resource.qrc"/>
	</resources>
	<connections/>
</ui>
